internal/pkg/runtime/engines/imgbuild/create_linux.go

```diff
@@ -183,8 +184,14 @@ func (engine *EngineOperations) CreateContainer(pid int, rpcConn net.Conn) error
 }
 
 func (engine *EngineOperations) copyFiles() error {
+	files := types.Files{}
+	for _, f := range engine.EngineConfig.Recipe.BuildData.Files {
+		if f.Args == "" {
+			files = f
+		}
+	}
 	// iterate through filetransfers
-	for _, transfer := range engine.EngineConfig.Recipe.BuildData.Files {
+	for _, transfer := range files.Files {
 		// sanity
 		if transfer.Src == "" {
 			sylog.Warningf("Attempt to copy file with no name...")
```

Comment:
Is there more to this warning message? I usually use `...` when I want to add something later.